pennylane/transforms/optimization/optimization_utils.py

```diff
@@ -79,6 +121,41 @@ def _fuse(angles_1, angles_2):
     return stack([z1, y, z2])
 
 
+def _fuse(angles_1, angles_2):
+    """Perform fusion of two angle sets. Separated out so we can do JIT with conditionals."""
+    # Compute the product of the quaternions
+    q = _quaternion_product(_zyz_to_quat(angles_1), _zyz_to_quat(angles_2))
```

Comment:
Any reason not to use unpacking here as before? IMO keeping the names `qx`, `qy` would boost readability here (they appear in a comment later down).

Reply:
Yes, you're right. I was thinking about the number of variables to pass on to `_regular/singular_quat_to_zyz(_jax)` but I guess 5 arguments are still fine :)